### ISSUE TITLE
1.14 image bug

### ISSUE THREAD

**Issue by GamerDaniel3000** [FaigerSYS/MapImageEngine#59]
Hey since 1.14 there is this bug that the images will probably be incorrectly cashed.
If you should know why it could fix it?
![Screenshot (1606)](https://user-images.githubusercontent.com/29311204/71320662-c36f9280-24ae-11ea-8b0a-a0b2aa7253be.png)

**Comment by Invy55** [comment]
Same problem here, please fix.

**Comment by GamerDaniel3000** [comment]
Ok blame @ mojang

**Comment by Invy55** [comment]
I was thinking, maybe creating the maps with differents name can help

**Comment by SuperAdam47** [comment]
+1

**Comment by simonzeimke06** [comment]
hello when i do /mie place (name) i can see the picture but i rejoin i  dont see  the picture only the bug frames can you say what i can do?? please

**Comment by Invy55** [comment]
Apparently that's a bug with packets so only the creator of the plugin can help us...

**Comment by simonzeimke06** [comment]
Owner of this plugin pls help us!

**Comment by SleepSpace9** [comment]
Seems more likely to be a client bug, so I'd say "MOJANG pls help us!"
Add a vote to the bug at their issue tracker:
https://bugs.mojang.com/browse/MCPE-54228
(idk if this will speed up a fix, but let's hope for the best)

**Comment by MisledPaul58** [comment]
Same ;(

**Comment by MisledPaul58** [comment]
Oh, why your chat looks like if you're playing in java edition?

**Comment by Invy55** [comment]
> Oh, why your chat looks like if you're playing in java edition?

Maybe he changed the interface to the other type (not pocket)

**Comment by Invy55** [comment]
> Seems more likely to be a client bug, so I'd say "MOJANG pls help us!"
> Add a vote to the bug at their issue tracker:
> https://bugs.mojang.com/browse/MCPE-54228
> (idk if this will speed up a fix, but let's hope for the best)

Yes, I confirm it's a cache bug infact if you stop and restart your minecraft client at the first join you can see the maps some times

**Comment by SuperAdam47** [comment]
@SleepSpace9 you how to fixed this bug
can you share with us?

**Comment by SleepSpace9** [comment]
My "fix" is more a workaround and sadly it doesn't work very reliably, so I didn't want to shared it yet. I'm waiting for a proper fix in the client.

**Comment by KingOfTurkey38** [comment]
@SleepSpace9 can you please share that "fix" with us, so we don't have players complaining about bad looking purple & black images .

**Comment by SleepSpace9** [comment]
Let's wait and see if the bug is fixed in the upcoming 1.16 release.

**Comment by SleepSpace9** [comment]
I would say it's fixed in 1.16, can others confirm that?
